src/components/App.jsx

```diff
@@ -5,7 +5,7 @@ import flow from 'lodash/flow'
 import { useClient } from 'cozy-client'
 import { RealTimeQueries } from 'cozy-client'
 
-import flag, { FlagSwitcher } from 'cozy-flags'
+import FlagSwitcher from 'cozy-flags/dist/FlagSwitcher'
```

Comment:
why don't we use something like

import { FlagSwitcher } from 'cozy-flags'

Reply:
parce qu'il n'est pas exposé https://github.com/cozy/cozy-libs/blob/master/packages/cozy-flags/src/index.js ?